dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsHelper.java

```diff
@@ -700,4 +711,84 @@ private boolean isAllFilledWithAsters(final char [] chars){
          return true;
     }
 
+    /**
+     *
+     * @param form
+     * @param user
+     * @return
+     * @throws DotSecurityException
+     * @throws IOException
+     * @throws DotDataException
+     */
+    StreamingOutput exportSecrets(final ExportSecretForm form, final User user)
+            throws DotSecurityException, IOException, DotDataException {
+
+        if(UtilMethods.isNotSet(form.getPassword())){
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'UtilMethods.isNotSet' due to existing static import 'com.dotmarketing.util.UtilMethods.isNotSet'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5931733)